website/docs/core-concepts/connecting-to-data-sources/README.md

```diff
@@ -1,5 +1,41 @@
 # Connecting to Datasource
 
+## **Outline**
+
+**Introduction** 
+
+**Prerequisites​**
+
+Before you begin, you'll need to create an account on Appsmith Cloud, or you can self-host Appsmith locally or on a hosting platform of your choice.
+
+**How to connect to a datasource**
+
+Explain how to connect to a datasource
```

Comment:
@harshilp24 could you also give a brief about what is the plan for this section.

Reply:
@harshilp24 - This would be a part of the API section, right? This section is a part of the datasource, which would be about databases, as you have segregated the content between two heads - API and Datasource.

Reply:
Something like this @jnikhila @danciaclara 
<img width="1070" alt="Screenshot 2022-12-05 at 6 59 01 PM" src="https://user-images.githubusercontent.com/43263619/205648976-a0bc018f-000a-419f-af54-24d28c9a86f3.png">

Reply:
ok, but we already have this info under Database and API separately, too, right? and that will make it seem redundant. Let this landing page be only about information that is common to both. things we want our users to know in general about datasources,  how they are global to that org, etc...  this page will primarily have the cards to those subpages and other shared information. 

I think for this ticket, we should also take into account the subpages - Databases and REST APIs and the data there.

Reply:
noted! will remove the how to connect section, also @danciaclara can you check this preview - https://appsmith-docs-gl03cdez4-get-appsmith.vercel.app/core-concepts/connecting-to-data-sources/,